src/Components/Header.jsx

```diff
@@ -1,9 +1,19 @@
+import { useNavigate } from "react-router-dom";
 import "./style/Header.css";
 
 const Header = () => {
+  const navigate = useNavigate();
+
+  const handleClick = (event) => {
+    event.preventDefault();
+    navigate("/reviews");
+  };
+
   return (
     <header id="main-header">
-      <h1 id="main-title">NC GAMES</h1>
+      <h1 id="main-title" onClick={handleClick}>
+        NC GAMES
+      </h1>
```

Comment:
We could wrap this in a `Link` tag rather than using `useNavigate` which would save us having to write a `handleClick`